src/go/rpk/pkg/config/config_test.go

```diff
@@ -203,14 +203,12 @@ func TestDefaultConfig(t *testing.T) {
 	expected := Config{
 		ConfigFile: "/etc/redpanda/redpanda.yaml",
 		Redpanda: &RedpandaConfig{
```

Comment:
minor :
Should this struct just be a const on the package level? 
```go
DEFAULT_R_CONFIG= RedpandaConfig{...}
```
That way it can be reused in tests & prod without having to remember to change it in two places ( which will definitely be a source of errors as more people contribute )

Reply:
There's already a config.DefaultConfig function.
Go doesn't support consts of struct instances.